### ISSUE TITLE
Vagrant can't use the requested machine because it is locked!

### ISSUE THREAD

**Issue by Spamme1** [mitchellh/vagrant#7968]
### Vagrant version
1.8.6

### Host operating system
Ubuntu 14.04.5 LTS

### Guest operating system
Windows 2012 SP1

### Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "a-mcf/Win2012R2-WMF5-min"
  config.vm.network "public_network", bridge: "eth0"

  config.vm.provider "virtualbox" do |vb|
    $name=File.basename(Dir.getwd)
    vb.name = $name+" "+Time.now.strftime("%a %d.%m.%Y %H-%M-%S")
    vb.linked_clone = true
    vb.memory = "4096"

    $port=5000
    for i in 0..$name.length-1
      $port+=$name[i].ord
    end
    vb.customize ["modifyvm", :id, "--vrdeport", $port]
  end

  config.vm.provision "shell", inline: <<-POWERSHELL
    tzutil /s "W. Europe Standard Time"
POWERSHELL
```

### Debug output
https://gist.github.com/Spamme1/193c143aa8a03ec3299a18d4b005518c

### Expected behavior
The virtual machine is initialized correctly and if it can't be initialized, it logs important information, why it couldn't be initialized. (Why the requested machine is locked? Which process is locking it?)

### Actual behavior
The new virtual machine couldn't be initialized because it is locked from "another vagrant process", what cannot be, because there isn't any other vagrant process, which uses this machine.
But this happens only sometime, it is not really something that can be reproduced every time.

### Steps to reproduce
I repeat myself, this doesn't happen regularly, it happens only sometimes. 
The process is triggered from a jenkins job, which uses a slave in the ubuntu server to execute a shell script with the following two commands: "vagrant destroy -f" and "vagrant up".

**Comment by Spamme1** [comment]
Another non really reproducible error in the provisioning of the virtual machine:

https://gist.github.com/Spamme1/758ca71fd7c1cda9904413697151a93c

**Comment by chrisroberts** [comment]
If this is a job that is triggered via jenkins, is it possible that it is getting triggered multiple times? That is my best guess without any viable way of reproducing your environment.

**Comment by Spamme1** [comment]
@chrisroberts Not really, because it is a nightly build (2:00 am) and in jenkins it appears to be triggered only once. When I arrive in office in the morning, I have sometimes an email in my inbox that the nightly build job failed, I just trigger it manually and it works.

**Comment by chrisroberts** [comment]
The first error in the original issue appears to be caused by two process attempting to make modifications. The second error (here: https://github.com/mitchellh/vagrant/issues/7968#issuecomment-263823258) looks to be a configuration issue within the box itself causing the error (winrm returning `Access is denied.`). If the errors are still occurring and box configuration is confirmed to be correct, please either re-open this issue or create a new issue. Thanks!
